test/special/tslint.js

```diff
@@ -11,36 +11,20 @@ const testCases = [
     expected: [],
   },
   {
-    name: 'detect specific plugins',
+    name: 'skip single built-in config',
     content: {
-      plugins: ['mocha'],
-    },
-    expected: [
-      'tslint-plugin-mocha',
-    ],
-  },
-  {
-    name: 'handle tslint config with short name',
-    content: {
-      extends: 'preset',
-    },
-    expected: [
-      'tslint-config-preset',
-    ],
-  },
-  {
-    name: 'handle tslint config with full name',
-    content: {
-      extends: 'tslint-config-preset',
+      extends: 'tslint:recommended',
     },
-    expected: [
-      'tslint-config-preset',
-    ],
+    expected: [],
   },
   {
-    name: 'skip tslint recommended config',
+    name: 'skip built-in configs',
     content: {
-      extends: 'tslint:recommended',
+      extends: [
+        'tslint:recommended',
+        'tslint:latest',
+        'tslint:foo',
+      ],
     },
     expected: [],
```

Comment:
I am thinking about if we should declare `tslint` in `expected` array. I am not sure if there is a user case to declare the `tslint.json` but not `script` CLI command. I personally think it is worth to declare `tslint` here.

Reply:
just to clarify, does this mean the tslint parser would return `['tslint']` every time (at least)?